database/src/migrations/20221222000000_bceid_basic_distinction.ts

```diff
@@ -0,0 +1,44 @@
+import { Knex } from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+const DB_SCHEMA_DAPI_V1 = process.env.DB_SCHEMA_DAPI_V1;
+
+/**
+ * Change 'BCEID' to 'BCEIDBASIC'
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    SET SCHEMA '${DB_SCHEMA}';
+    SET SEARCH_PATH = ${DB_SCHEMA}, ${DB_SCHEMA_DAPI_V1};
+
+    UPDATE
+      user_identity_source
+    SET
+      name = 'BCEIDBASIC',
+      description = 'BCEID BASIC user source system.'
```

Comment:
Looks like this migration is inserting the basic value, while the business one is being added directly to the populate_user_identity_source file. We can probably remove this migration, and just add this value to the populate file directly.

We will need a proper migration in SIMS, but for biohub we can get away with modifying the existing migrations since we aren't in PROD yet.

Reply:
Ahhh hadn't realized we could change the populate script. Changed.